prov/verbs/src/verbs_ep.c

```diff
@@ -318,14 +318,23 @@ static int fi_ibv_ep_bind(struct fid *fid, struct fid *bfid, uint64_t flags)
 			break;
 		case FI_CLASS_EQ:
 			ep->eq = container_of(bfid, struct fi_ibv_eq, eq_fid.fid);
+
+			/* Make sure EQ channel is not polled during migrate */
+			fastlock_acquire(&ep->eq->lock);
```

Comment:
Locking granularity question: Is it necessary to lock the entire EQ, or could be put a lock around RDMA CM to prevent the EQ poll from interacting with RDMA CM during a migration, but let other activities continue to advance without such a huge lock? Silly question probably considering I *think* the EQ only addresses connection management at this point, but I thought I'd ask.

Reply:
Right it is only connection management at this time. The use of the EQ lock around reading the event channel was already in place, so I think creating a new lock would effectively result in the same level of coarseness and require the read of the channel take two locks.